weblate/settings_example.py

```diff
@@ -530,6 +531,13 @@
 # tmserver URL
 MT_TMSERVER = None
 
+# SAP Translation Hub
+MT_SAP_TRANSLATION_HUB_BASE_URL = None
+MT_SAP_TRANSLATION_HUB_SANDBOX_APIKEY = None
+MT_SAP_TRANSLATION_HUB_USERNAME = None
+MT_SAP_TRANSLATION_HUB_PASSWORD = None
+MT_SAP_TRANSLATION_HUB_USE_MT = True
```

Comment:
These should be added to `weblate/trans/models/conf.py` in order to have default value if user doesn't specify them in the `settings.py`.

Reply:
added this as you requested